fic_modules/hg.py

```diff
@@ -302,10 +306,12 @@ def extract_json_from_hg(json_files, path_to_files, days_to_generate):
                                         review,
                                         commit_date)
             except AttributeError:
-                logger.info("Attribute Error!! \n "
-                            "Probable issue is an malfunctioned json file.. "
-                            "Please check the following file:{}".format(file))
+                LOGGER.exception("Attribute Error!! \n Probable issue is an "
```

Comment:
Typo here: Probable issue is an to `a` as the following word doesn't start with a vocal.

Reply:
Fixed.